### PR TITLE
[Merged by Bors] - feat(Analysis/Analytic/Basic): add missing `AnalyticOn.neg` lemma

### PR DESCRIPTION
**Pull request by Command-Master**
Add missing `AnalyticOn.neg` lemma.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/Analytic/Basic.lean [hunk 1]
```diff
@@ -617,6 +617,10 @@ theorem AnalyticOn.add {s : Set E} (hf : AnalyticOn 𝕜 f s) (hg : AnalyticOn 
   fun z hz => (hf z hz).add (hg z hz)
 #align analytic_on.add AnalyticOn.add
 
+theorem AnalyticOn.neg {s : Set E} (hf : AnalyticOn 𝕜 f s) :
+    AnalyticOn 𝕜 (-f) s :=
+  fun z hz => (hf z hz).neg
```
Comment (grunweg) [inline Mathlib/Analysis/Analytic/Basic.lean:622]: Can you put the conclusion on the same line as the hypotheses, and use "\maps" instead of "=>"? Thanks!

**Review by grunweg — COMMENTED** [review]
Thanks for your PR. I have one minor comment; otherwise, looks good to me!

**Comment by grunweg** [comment]
Thanks, looks good now!
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**